build/buildx/drivers/remote.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Remote driver"
+keywords: build, buildx, driver, builder, remote
+fetch_remote:
+  line_start: 2
+  line_end: -1
+---
+
+> Technical preview
```

Comment:
Also, should this be **Beta** and not Technical Preview so we are consistent with our terminology? See https://deploy-preview-14950--docsdocker.netlify.app/release-lifecycle/.

Reply:
Ah thanks yes!

Reply:
Looking at the statement, "Can be discontinued without notice" I'm not sure this is the case for the remote driver as this will still be released but not yet available.

I think we could also have "Technical preview" in this page if it makes sense:

> **Technical preview** means a service or software that is under active development and in stages of testing. Its purpose is to establish a feedback loop, allowing customers to influence its development and direction.

But in the meantime I will use Beta terminology.

Reply:
Yes, generally, we should avoid forward-looking statements in Docs. I guess we can tag it as Beta for now and update this page to remove the Beta flag when the remote drivers features goes GA.